src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -768,11 +779,25 @@ def _process_corpus_crashes(context, result):
     task_creation.create_tasks(testcase)
 
 
-def _get_cross_pollinate_fuzzers(engine_name, current_fuzzer_name):
-  """Return a list of fuzzer objects to use for cross pollination."""
-  cross_pollinate_fuzzers = []
-
+def _select_targets_and_jobs_for_pollination(engine_name, current_fuzzer_name,
+                                             method):
+  """Select jobs to use for cross pollination."""
   target_jobs = list(fuzz_target_utils.get_fuzz_target_jobs(engine=engine_name))
+  similar_tagged_targets = corpus_tagging.get_similarly_tagged_fuzzers(
+      current_fuzzer_name)
+  # Cross Pollinate randomly if there are no similarly tagged targets.
+  if similar_tagged_targets:
```

Comment:
you can just do
`if similar_tagged_targets and method == 'tagged'` 

here to avoid an indentation level.